src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,476 @@
+.format-button-login
+ {
+    width:auto;
+    color: white;
+    padding: 14px 20px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 50px;
+    cursor: pointer;
+    width: 100%;
```

Comment:
There is two declarations of width values in this class, please remove one of them